cockroachcloud/pci-dss.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Payment Card Industry Data Security Standard (PCI DSS) Compliance in CockroachDB Dedicated
+summary: Learn about compliance with the Payment Card Industry Data Security Standard (PCI DSS) for CockroachDB Dedicated clusters.
+toc: true
+docs_area: manage.security
+---
+
+The Payment Card Industry Data Security Standard (PCI DSS) is a minimum set of requirements for the safe handling of sensitive data associated with credit and debit cards. In the PCI DSS standard, this data is referred to as "cardholder data". When implemented correctly, PCI DSS helps to protect cardholder data from fraud, exfiltration, and theft. PCI DSS is mandated by credit card issuers but administered by the [Payment Card Industry Security Standards Council](https://www.pcisecuritystandards.org/).
+
+Many organizations who do not store cardholder data still rely upon compliance with PCI DSS to help protect other sensitive or confidential data or metadata.
+
+Responsibility for compliance with PCI DSS is shared among multiple parties, including card issuers, banks, software-as-a-service (SaaS) providers, and retail merchants. Compliance measures are implemented as a series of business practices, security controls, and technological solutions. An organization's compliance with PCI DSS is certified by a PCI Qualified Security Assessor (QSA).
```

Comment:
```suggestion
Responsibility for compliance with PCI DSS is shared among multiple parties, including card issuers, banks, software as a service (SaaS) providers, and retail merchants. Compliance measures are implemented as a series of business practices, security controls, and technological solutions. An organization's compliance with PCI DSS is certified by a PCI Qualified Security Assessor (QSA).
```

Reply:
I don't think this is correct, because "software-as-a-service" is an adjective here, modifying "providers".